pkg/storage/system_intake.go

```diff
@@ -400,7 +397,7 @@ func (s *Store) FetchSystemIntakeByLifecycleID(ctx context.Context, lifecycleID
 	intakes := []models.SystemIntake{}
 	const matchClause = `
 		WHERE system_intakes.lcid=$1
-			AND system_intakes.archived_at IS NULL AND system_intakes.status != 'WITHDRAWN'
```

Comment:
double check

Reply:
removed a hard-coded conditional in SQL - is this ok?

Reply:
Good news - this Store method's not even used anywhere, so we can just delete this if you'd like!